index.js

```diff
@@ -3,6 +3,8 @@ const { default: ConnectionString } = require('mongodb-connection-string-url');
 const ATLAS_REGEX = /\.mongodb(-dev)?\.net$/i;
 const LOCALHOST_REGEX = /^(localhost|127\.([01]?[0-9][0-9]?|2[0-4][0-9]|25[0-5])\.([01]?[0-9][0-9]?|2[0-4][0-9]|25[0-5])\.([01]?[0-9][0-9]?|2[0-4][0-9]|25[0-5])|0\.0\.0\.0|(?:0*\:)*?:?0*1)$/i;
 const DIGITAL_OCEAN_REGEX = /\.mongo\.ondigitalocean\.com$/i;
+const COSMOS_DB_REGEX = /mongo\.cosmos\.azure\.com$/i;
```

Comment:
```suggestion
const COSMOS_DB_REGEX = /\.cosmos\.azure\.com$/i;
```

Probably any `cosmos.azure.com` domain will be a CosmosDB instance?

Reply:
Yea that true, I will take that. Thank you :)